django_socio_grpc/services/servicer_proxy.py

```diff
@@ -197,10 +203,15 @@ async def handler(request: Message, context) -> Awaitable[Message]:
             request_container = GRPCRequestContainer(
                 request, proxy_context, action, service_instance
             )
-            response = await safe_async_response(
-                self._middleware_chain, request_container, self.async_process_exception
-            )
-            return response.grpc_response
+            try:
+                exc = None
+                response = await safe_async_response(self._middleware_chain, request_container)
```

Comment:
safe_async_response look like it is not using process_exceptions arguments. Should be deleted then